package.json

```diff
@@ -166,7 +166,9 @@
     "@types/react-router-dom": "^5.1.7",
     "axios": "^0.21.1",
     "bootstrap": "^4.6.0",
+    "dotenv": "^10.0.0",
```

Comment:
This might also be a dev dependency. The Netlify functions example code suggests it is not needed in production I think  https://www.netlify.com/products/functions/

Reply:
This might even not be necessary at all, since netlify also provides env variables when serving at localhost